src/Compilers/Core/Portable/DiagnosticAnalyzer/SuppressMessageAttributeState.cs

```diff
@@ -108,49 +108,8 @@ public bool IsDiagnosticSuppressed(Diagnostic diagnostic, out AttributeData supp
             return false;
         }
 
-        private bool IsDiagnosticSuppressed(Diagnostic diagnostic, out SuppressMessageInfo info, ISymbol symbolOpt = null)
```

Comment:
there are no callers of this method that pass in "symbolOpt".  So symbolOpt is always null. So nothing ever called `IsDiagnosticSuppressed(diagnostic.Id, symbolOpt, out info)`.